lib/src/chips_input.dart

```diff
@@ -76,6 +77,7 @@ class ChipsInput<T> extends StatefulWidget {
   final bool autofocus;
   final bool allowChipEditing;
   final FocusNode focusNode;
+  final List<dynamic> initialSuggestions;
```

Comment:
Shouldn't this be a `List<T>` instead of `List<dynamic>`?  (Whenever I see `dynamic`, it is often a _code smell_.  In general, we want to promote strong typing so that Dart is effective.)

Reply:
Also, there should be some comment for this new field.  _What is it used for?_ Is _null_ acceptable?